src/config/dynamic-theme-fixes.config

```diff
@@ -1791,7 +1791,10 @@ code:first-of-type {
 .sc-186t4q4-1 {
     background-color: ${rgba(3, 9, 15, 0.02)} !important;
 }
-#ak-main-content {
+div.zgsxji-0.jMIyWM > div > div > div > div {
+    background-color: var(--darkreader-bg--ds-surface, #181a1b) !important;
```

Comment:
```suggestion
    background-color: var(--darkreader-bg--ds-surface, var(--darkreader-neutral-background)) !important;
```

Reply:
This makes more sense!

Reply:
I had to rever this, as it wasn't behaving as expected.
<img width="580" alt="image" src="https://user-images.githubusercontent.com/1766174/190862164-43432c33-358f-494c-8c9b-71dd8abac043.png">

vs. 

<img width="564" alt="image" src="https://user-images.githubusercontent.com/1766174/190862284-88ed9556-7a75-480f-9669-15b8d6418c44.png">